service/src/main/scala/org/apache/celeborn/server/common/http/authentication/AuthenticationFilter.scala

```diff
@@ -64,6 +66,13 @@ class AuthenticationFilter(conf: CelebornConf, serviceName: String) extends Filt
     case Service.WORKER => conf.get(CelebornConf.WORKER_HTTP_PROXY_CLIENT_IP_HEADER)
   }
 
+  private val administrators: Set[String] = serviceName match {
+    case Service.MASTER =>
+      conf.get(CelebornConf.MASTER_HTTP_AUTH_ADMINISTERS).toSet + Utils.currentUser
```

Comment:
Why add UserGroupInformation.getCurrentUser.getShortUserName to administrator list?

Reply:
thanks, after think twice, I remove it.

Due that for celeborn, kerberos is not common enabled, and the server might be launched with normal linux user.
To prevent the admin privileges leak, it is better not Implicit add the system property `user.name` into administrator list.